src/main/resources/example.conf

```diff
@@ -25,6 +25,7 @@ bwa.executable   = ${pipeline_dir}/bwa/bwa
 bwa-kit.dir      = ${pipeline_dir}/bwa.kit
 tabix.executable = ${pipeline_dir}/tabix
 verifybamid.executable = ${pipeline_dir}/verifyBamID
+samtools.executable = ${pipeline_dir}/samtools/samtools
```

Comment:
I have a samtools dir with a samtools executble (and others) in it, but this is just the example anyway.